app/src/components/v-image.vue

```diff
@@ -68,7 +68,14 @@ async function loadImage() {
 		const base64 = window.btoa(raw);
 		srcData.value = `data:${contentType};base64,${base64}`;
 	} catch (err) {
-		emit('error', err);
+		imgError.value = true;
+		var decodedString = String.fromCharCode.apply(null, new Uint8Array(err.response.data));
+		var obj = JSON.parse(decodedString);
+
+		if (obj && obj.errors && obj.errors.length && obj.errors[0].message) {
+			const message = obj.errors[0].message;
+			emit('error', message);
```

Comment:
I think it would be better to call the `emit` statement regardless of the availability of error messages as it used to be before.